js/genkit/src/evaluator.ts

```diff
@@ -14,4 +14,23 @@
  * limitations under the License.
  */
 
-export * from '@genkit-ai/ai/evaluator';
+export {
```

Comment:
I'm a little out of the loop on evaluators: Is there a defineEvaluator method? If so does it need to be exported

Reply:
yes, there is. it's `ai.defineEvaluator` now.